btree/btree.c

```diff
@@ -3740,6 +3767,58 @@ int64_t btree_open_tick(struct m0_sm_op *smop)
 	}
 }
 
+/**
+ * btree_close_tree_tick function is used to traverse through different states
+ * to facilitate the working of m0_btree_close().
+ *
+ * @param smop     represents the state machine operation
+ * @return int64_t returns the next state to be executed.
+ */
+int64_t btree_close_tree_tick(struct m0_sm_op *smop)
+{
+	struct m0_btree_op    *bop     = M0_AMB(bop, smop, bo_op);
+	struct m0_btree       *tree    = bop->bo_arbor;
+	struct td             *td_curr = tree->t_desc;
```

Comment:
Remove line 3780 and directly take `bop->bo_arbor->t_desc`. I do not see `tree` getting anywhere else in the code.

Reply:
Ok. I will remove it.

Reply:
Updated.